### PR TITLE
#6721: Fix log nan issue

### PR DESCRIPTION
**Pull request by ruthreshx**
### Ticket
[Link to Github Issue](https://github.com/tenstorrent/tt-metal/issues/6721)

### What's changed
Fix log nan issue

### Checklist
- [x] Post commit CI passes
- [x] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)

### CODE REVIEW

File: tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_sfpu/ckernel_sfpu_log.h [hunk 1]
```diff
@@ -61,6 +61,11 @@ sfpi_inline void calculate_log_body(const uint log_base_scale_factor) {
     }
     v_endif;
 
+    v_if (in < 0.0F) {
+        result = std::numeric_limits<float>::quiet_NaN();
+    }
+    v_endif;
+
```
Comment (rtawfik01) [inline tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_sfpu/ckernel_sfpu_log.h:68]: @ruthreshx Can you run the performance pipeline as well and make sure the extra conditional cycles have not had large effects on performance?

Comment (ruthreshx) [reply tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_sfpu/ckernel_sfpu_log.h:68]: Hi @rtawfik01 ,
May I know what all the workflows that needs to be run for performance check?

Comment (rtawfik01) [reply tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_sfpu/ckernel_sfpu_log.h:68]: This one for example: https://github.com/tenstorrent/tt-metal/actions/workflows/perf-device-models.yaml

**Comment by VirdhatchaniKN** [comment]
Closing PR as issue has been assigned to LLK Team, https://github.com/tenstorrent/tt-metal/issues/12776

**Pull request closed without merging**